README.md

```diff
@@ -25,6 +25,13 @@ go run effx.go event --title "title" --message "meassage" --service "dooku" --ta
 go run effx.go sync -d . -k ${EFFX_API_KEY} --disable-languauge-detection
 ```
 
+### configure automatic service detection
+you can configure which directory names can 
+automatically detected services. For example any directories inside `services/` or `apps/` will contain detected services
```

Comment:
have you tested for nested directories? That will be a common pattern

Reply:
yeah I added a test